cmd/utils/okteto.go

```diff
@@ -91,3 +91,26 @@ func ShouldCreateNamespace(ctx context.Context, ns string) (bool, error) {
 	}
 	return false, nil
 }
+
+// ShouldCreateNamespace checks if the user has access to the namespace.
+// If not, ask the user if he wants to create it
+func ShouldCreateNamespaceStateless(ctx context.Context, ns string, c *okteto.OktetoClient) (bool, error) {
```

Comment:
Also, we could flag the func as deprecated if we want to promote the usage of the new one. WDYT?

Reply:
If we create the client inside the function we can't test it given that it will always create the real client. I think it's ok to pass the client as argument or a provider

Reply:
great, I've refactor the code to reuse one inside other